cli/src/command.rs

```diff
@@ -477,13 +484,19 @@ pub fn run() -> Result<()> {
 			#[cfg(not(feature = "polkadot-native"))]
 			panic!("No runtime feature (polkadot, kusama, westend, rococo) is enabled")
 		},
-		#[cfg(not(debug_assertions))]
 		Some(Subcommand::HostPerfCheck) => {
 			let mut builder = sc_cli::LoggerBuilder::new("");
 			builder.with_colors(true);
 			let _ = builder.init();
 
-			host_perf_check()
+			#[cfg(not(debug_assertions))]
+			{
+				host_perf_check()
+			}
+			#[cfg(debug_assertions)]
```

Comment:
FYI, `debug_assertions` != `debug`.

You can do this to get the build type: https://github.com/paritytech/substrate/blob/master/client/consensus/slots/build.rs#L23

Reply:
Thank you! It bothered me too, feels weird that Rust doesn't have it in std